sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ConnectionStringProperties.java

```diff
@@ -83,25 +90,34 @@ public ConnectionStringProperties(String connectionString) {
                 sharedAccessKeyValue = value;
             } else if (key.equalsIgnoreCase(ENTITY_PATH)) {
                 entityPath = value;
+            } else if (key.equalsIgnoreCase(SHARED_ACCESS_SIGNATURE)
+                && value.toLowerCase(Locale.getDefault()).startsWith(SAS_VALUE_PREFIX)) {
```

Comment:
This value should be culture agnostic, I would use the culture invariant locale.